src/ecosystem/index.rst

```diff
@@ -0,0 +1,14 @@
+
+======================================
+Ecosystem
+======================================
+
+Nextstrain project has grown and a few projects have have spun-off of it.
+
+
+.. toctree::
+   :maxdepth: 2
+   :titlesonly:
+   :caption: Projects in Nextstrain ecosystem
+
+   Nextclade: analysis of viral genomes <https://docs.nextstrain.org/projects/cli/en/latest/>
```

Comment:
The URL points to the Nextstrain CLI subproject.

Reply:
Sorry. Coding late.